LearningMissionSimulation/AttributeGenerator.cs

```diff
@@ -269,7 +269,7 @@ public static string GetUsername()
         public static string GetPassword(int minLength, int maxLength)
         {
             int count = random.Next(minLength, maxLength);
```

Comment:
If minLength equals to maxLength, your method will throw an exception. Please fix.